enhancements/sig-architecture/6-placements/README.md

```diff
@@ -0,0 +1,550 @@
+# New Placement APIs
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [website](https://github.com/open-cluster-management/website/)
+
+## Summary
+
+The proposed work would define new placement APIs to select a set of managed clusters from multiple ManagedClusterSets.
+
+## Motivation
+
+Since placementrule is used in multiple components (GRC/App/Observability). It is more appropriate to define it in API repo and have a generic component to consume it based on the current placementrule APIs in https://github.com/open-cluster-management/multicloud-operators-placementrule. 
+
+The placement APIs will provide a definition on how to select a set of managed clusters in multiple ManagedClusterSets. And placement decisions will be generated according to the spec defined in the placement APIs.
+
+Placement APIs can be used by a higher primitive and controller to decide which clusters the manifests should be placed.  
+
+### User Stories
+
+#### Story 1: A cluster-admin on hub can place a policy or workload to all clusters in all ManagedClusterSets; The cluster-admin on hub can also place a policy or workload to any selected clusters in ManagedClusterSets.
+
+#### Story 2: A normal user on hub can place a policy or a workload to a slice of clusters in one or multiple ManagedClusterSets that the user is authorized to access.
+
+#### Story 3: A user can use predicate to select clusters
+  - Select openshift clusters only
+  - Select clusters in aws and gcp
+  - Select cluster not in vsphere
+  - Select openshift clusters with a specific version
```

Comment:
Should the spec be broader and target Kubernetes clusters in general (e.g. "Select Kubernetes clusters with a specific version" and "Select Kubernetes clusters based on a specified flavor - e.g. OpenShift"

Reply:
Updated as suggested.